src/index.spec.ts

```diff
@@ -255,7 +263,7 @@ describe('EppoJSClient E2E test', () => {
       return mockObfuscatedUfcFlagConfig;
     });
     const subjectAttributes = { foo: 3 };
-    globalClient.setLogger(mockLogger);
+    globalClient.setAssignmentLogger(mockLogger);
```

Comment:
`setLogger()` has been renamed `setAssignmentLogger()` to better differentiate it from the newly-added `setBanditLogger()`